### ISSUE TITLE
Tiny-yolov4 + DeepSort 

### ISSUE THREAD

**Issue by MuhammadAsadJaved** [yehengchen/Object-Detection-and-Tracking#79]
Hi, 
      1-Is there any way to use Tiny-yolov4 + deepSort?  
       2- in the NVIDIA NX yolov3+DeepSort is very slow. Is it possible to use TensorRT to optimize yolov3 or v4 and then combine with DeepSort to make it faster? 
       3-Any other suggestion to run it faster in NX?

**Comment by ewuxjun** [comment]
Hi，i use the tensorrt to test yolov3(tensorrt)+deepsort , the performance is 5fps, what about you ?

**Comment by MuhammadAsadJaved** [comment]
@ewuxjun  Hi buddy,  I am trying to run it in NX, I still have some problems. Which project are you using for Jetson and which device are you using?

**Comment by ewuxjun** [comment]
maybe it is the same as your project, the device is also jetson Xavier nx

**Comment by MuhammadAsadJaved** [comment]
OH. Great, let me try then I will share the results with you.   BTW I am using this project,   

https://github.com/Stephenfang51/tracklite

I am confused about one thing, the only YOLOV3 TRT can run in real-time in NX. Some people said in the YOLO+DeepSort most of the time is consumed by YOLO so that's why YOLO+DeepSort is slow. But if we converted YOLO to TRT and then combined with DeepSort then why it is still very slow? Do you have any idea?

**Comment by ewuxjun** [comment]
yeah, I have encountered the same problem, yolov3 can run in real-time in Xavier nx ,as you say, yolov3+deepsort is so slow, I don't know these two modules influence each other, so now I am searching for resnet+deepsort, we just know yolov3 need much performance

**Comment by MuhammadAsadJaved** [comment]
@ewuxjun    let's try it. Is it possible to connect over WeChat as we are working on the same problem, it might be helpful? 

If Yes then my WeChat is: 13621297680

**Comment by ewuxjun** [comment]
ok, I have added friends, you can check it

**Comment by MuhammadAsadJaved** [comment]
Received. Thanks

**Comment by heleiqiu** [comment]
deep sort does not consume too much GPU, and many operations of the algorithm use CPU. However, the CPU performance of Jetson Xavier NX is worse than that of PC, and it is not fully utilized.
